Makefile

```diff
@@ -687,6 +687,17 @@ tidy_cpp : $(libceed.cpp:%=%.tidy)
 
 tidy : tidy_c tidy_cpp
 
+ifneq ($(wildcard ../iwyu/*),)
+  IWYU_DIR ?= ../iwyu
+endif
+ifneq ($(IWYU_DIR),)
+  IWYU_CC = $(IWYU_DIR)/build/bin/include-what-you-use
+
+  iwyu_cc :
+    CC = $(IWYU_CC)
+  iwyu : lib | iwyu_cc
+endif
```

Comment:
FWIW, I `yay -S include-what-you-use`, but `make iwyu` says there's nothing to do. And if I set `make IWYU_CC=include-what-you-use IWYU_DIR=dummy` then it doesn't even try to build the library.